web/src/main/webapp/WEB-INF/jsp/team/match/detail.jsp

```diff
@@ -0,0 +1,218 @@
+                        <span> </span>
+                        <strong><c:out value="${matchResult.match.awayTeam.name}"/></strong>
+                    </legend>
+                    <p class="m-a-0 text-md-center">
+                        <strong><span>Start: </span></strong>
+                        <c:out value="${matchResult.match.startTime.format(formatter)}"/>
+                    </p>
+                    <p class="text-md-center">
+                        <strong><span>End: </span></strong>
+                        <c:out value="${matchResult.match.endTime == null ? ' - ' : matchResult.match.endTime.format(formatter)}"/>
+                    </p>
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-md-6">
+                                <div class="table-div table-responsive">
+                                    <table class="table table-fixed table-hover table-striped table-fixed-small">
+                                        <thead>
+                                            <tr class="tr-fixed-small">
+                                                <th>Time</th>
+                                                <th>Scorer</th>
+                                                <th>Assist</th>
+                                                <th> </th>
+                                            </tr>
+                                        </thead>
+                                        <tbody class="tbody-scrollable-small">
+                                            <c:forEach items="${matchDetail.homeGoals}" var="homeGoal">
+                                                <tr class="tr-fixed-small">
+                                                    <td><c:out value="${homeGoal.matchTime.format(formatter)}"/></td>
+                                                    <td><c:out value="${homeGoal.scorer.firstname} ${homeGoal.scorer.surname}"/></td>
+                                                    <td><c:out value="${homeGoal.assistant.firstname} ${homeGoal.assistant.surname}"/></td>
+                                                    <sec:authorize access="hasAnyRole('ROLE_ADMIN', 'ROLE_MODERATOR')">
+                                                        <td>
+                                                            <a href="#deleteHomeGoalModal" title="Delete goal" class="btn btn-block btn-goal-delete" data-toggle="modal" role="button">
+                                                                <strong>X</strong>
+                                                            </a>
+                                                        </td>
+                                                        <div class="modal fade" id="deleteHomeGoalModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" style="display: none;">
```

Comment:
buďto to `aria-` vyplň něčím rozumným, nebo ten attribut smaž.. slouží to jako pomoc pro čtečky pro nevidomé